recipes/discorpy/meta.yaml

```
@@ -0,0 +1,42 @@
+package:
+  name: discorpy
+  version: 1.4
+
+source:
+  url: https://github.com/DiamondLightSource/discorpy/archive/refs/tags/v1.4.tar.gz
+  sha256: 4144dcad781312c8b2316d785c9e8110715b2fbc7f15180e1760b9191f0eb9c7
+build:  
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python >=3.7
+    - setuptools
+  run:
+    - python >=3.7
+    - matplotlib
```

Unless you need a PyQt-based GUI, better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```